tests/installation/handle_reboot.pm

```diff
@@ -22,6 +22,7 @@ use Utils::Backends 'is_remote_backend';
 use utils qw(reconnect_mgmt_console);
 
 sub run {
+    return if (check_var('VIRSH_VMM_TYPE', 'linux'));
```

Comment:
why to schedule this module if we are going just to return?

Reply:
Yeah, I agree, this is looks as a controversial decision. But in case it will not be here, the extra schedule for xen-pv should be created, and the only difference there from other backends will be the absence of `handle_reboot` module. 

I reasoned like this: `handle_reboot` module controls reboot process and even if reboot does not need any special actions on some particular backend, decision of skipping it may be made also in `handle_reboot`.

But, if it seems confusing, I can remove the line and create separate schedule for xen-pv insead. WDYT?

Reply:
for me sound better with a different schedule, when maintaining that code in the future nobody will need to remember that reasoning.

Reply:
Updated. Also this change require MR to be merged in scheduling repo.